content/en/docs/armory-admin/evaluate-artifacts-stage-enable.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Enable the Evaluate Artifacts Stage Plugin
+description: >
+  Enable the Evaluate Artifacts Stage plugin to add a stage to Armory Enterprise that makes it easy to evaluate SpEL expressions inside of Spinnaker™ artifacts.
+aliases:
+  - /docs/plugin-guide/plugin-evaluate-artifacts/
+---
+
+![Proprietary](/images/proprietary.svg)
+
+## Overview
+
+Armory’s Evaluate Artifacts plugin allows you to easily evaluate SpEL queries inside of any Spinnaker artifacts. Some artifact types and their associated stages, such as Kubernetes, support  using SpEL to read parameters from the Spinnaker context and inject them into an artifacts manifest at deploy time. Other stages, such as the Terraform Integration stage, do not support this out of the box. The Evaluate Artifacts Stage plugin adds supports for evaluating SpEL against any artifact within your Spinnaker pipeline.
+
+For information about how to use the stage, see [Use the Evaluate Artifacts Stage]({{< ref "evaluate-artifacts-plugin-use.md" >}}).
+
+## Setup
+
+Add the following snippet to your Operator config, such as `spinnakerservice.yml`:
+
+{{< prism lang=yaml line="4-5, 10, 22" >}}
+    spec:
+      spinnakerConfig:
+        profiles:
+          spinnaker:  
+            spinnaker:
+              extensibility:
+                plugins:
+                  Armory.EvaluateArtifactsPlugin:
+                    enabled: true
+                    version: <PLUGIN_VERSION> # Replace with the version you want to use
+              repositories:
+                evaluateArtifacts:
+                  url: https://raw.githubusercontent.com/armory-plugins/evaluate-artifacts-releases/master/repositories.json
+          gate:
+            spinnaker:
+              extensibility:
+                deck-proxy:
+                  enabled: true
+                  plugins:
+                    Armory.EvaluateArtifactsPlugin:
+                      enabled: true
+                      version: <PLUGIN_VERSION> # Replace with the version you want to use. Omit the 'v' when providing a version. For example, use 0.1.0, not v0.1.0
+{{< /prism >}}
+
+Keep the following in mind when using this configuration snippet: 
+
+* Make sure to include the nested `spinnaker` parameters on lines 4 and 5. Both are required because of how Operator configs are processed and consumed by Armory Enterprise.
```

Comment:
```suggestion
* Make sure to include the nested `spinnaker` parameters on lines 4 and 5. Both are required because of how Spinnaker consumes plugin configuration.
```